flocker/node/_deploy.py

```diff
@@ -757,23 +757,22 @@ def calculate_changes(self, desired_configuration, current_cluster_state):
         if desired_open_ports != set(self.network.enumerate_open_ports()):
             phases.append(OpenPorts(ports=desired_open_ports))
 
-        current_node_applications = set(
+        running_applications = set(
             app for app in current_node_state.applications if app.running)
         all_applications = current_node_state.applications
 
         # Compare the applications being changed by name only.  Other
         # configuration changes aren't important at this point.
-        current_state = {app.name for app in current_node_applications}
+        running = {app.name for app in running_applications}
         desired_local_state = {app.name for app in
                                desired_node_applications}
```

Comment:
:question: There is inconsistent naming of things having sets of names, vs `Applications` here, which makes reviewing the code slightly harder.